### DISCUSSION TITLE
What maxLevel option affects?

### DISCUSSION THREAD

**Discussion by zdm** [docsifyjs/docsify#1590] [Q&A] [answered]
Hi, 
It is unclear from documentation what is `maxLevel` option for?
`subMaxLevel` is for sidebar toc depth, but maxLevel?
I created various configurations and tried with various maxLevel values, but unable to found any differences.
Could somebody help to understand?

**Comment by sy-records** [comment, marked as answer]
If you do not configure `loadSidebar`, then by default all headings in the document will be grabbed and rendered as a table of contents. `maxLevel` can be configured to support the maximum level of headings to be rendered.

**Reply by trusktr** [reply]
Don't forget to mark answer. :)
